src/services/config/config-service.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from "node:fs"
+import * as path from "node:path"
+import { CLIError } from "@oclif/core/lib/errors"
+
+const DEFAULT_CONFIG_PATH = path.resolve(__dirname)
+const DEFAULT_CONFIG_FILE = "default-config.json"
+const CONFIG_FOLDER = ".ent"
```

Comment:
This is also used in `initializer.ts`. I propose to centralize this constant in a `Paths.ts` class as it will be used in other contexts